<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MFileChooser</class>
 <widget class="QWidget" name="MFileChooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>516</width>
    <height>115</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QLabel" name="lblDirTitle">
       <property name="text">
        <string>&amp;into directory</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
       <property name="buddy">
        <cstring>txtDir</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDir">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true"/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbEdit">
       <property name="toolTip">
        <string>edit manually</string>
       </property>
       <property name="text">
        <string>&amp;edit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btChooseDir">
       <property name="text">
        <string>&amp;choose</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frmInfo">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lblWarn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="text">
         <string>status</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="margin">
         <number>3</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btView">
        <property name="text">
         <string>&amp;go to</string>
        </property>
        <property name="icon">
         <iconset resource="resfile.qrc">
          <normaloff>:/new/icons/icons/go.png</normaloff>:/new/icons/icons/go.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="0" margin="0"/>
 <resources>
  <include location="resfile.qrc"/>
 </resources>
 <connections/>
</ui>
